docs: Added commands to README.md (#40)
DenverCoder1/dev-pro-tips-bot: README.md

* `!markdown` - displays Discord markdown tips

* `!ping` - check for a response from the bot

* `!help` - displays more info about commands

> * `!roll` - roles a given amount of dice in the format `_d_`

Reviewer: ```suggestion
* `!roll` - rolls a given amount of dice in the format `_d_`
```

* `!help` - displays more info about commands

> * `!roll` - roles a given amount of dice format `_d_`

Reviewer: ```suggestion
* `!roll` - roles a given amount of dice in the format `_d_`
```

* `!choose` - chooses an item randomly from a list

> * `!docs` - fetch various python library documentation use `!help docs` for more info

Reviewer: ```suggestion
* `!docs` - fetch documentation for various Python libraries. Use `!help docs` for more info.
```

* **Welcome** - Sends a welcome message to users and gives them the Unassigned role

* **Button Roles** - Assigns roles when users click the button components

* **YouTube Feed** - posts new videos from my YouTube channel in `#videos`

### Owner Commands

> * `!roles` - start a roles view, add or remove a role

Reviewer: ```suggestion
* `!roles` - start a role view for users to add and remove their roles
```

> * `!add_confirm` - start a confirm view, confirm role was added

Reviewer: ```suggestion
* `!add_confirm` - add a confirm button to a given message
```

### Owner Context Menu Commands

> * `giveaway` - start and create a giveaway

Reviewer: ```suggestion
* `giveaway` - start and select winners for a giveaway
```
